index.js

```diff
@@ -28,7 +30,7 @@ module.exports = function(str, options) {
   }
 
   var lines = str.match(re) || [];
-  var res = indent + lines.join(newline);
+  var res = indent + lines.map(function(str){return escape(str)}).join(newline);
```

Comment:
I was trying to figure out how to make that map call look better--somehow it didn't occur to me that it was already a function!